crates/api_models/src/payment_methods.rs

```diff
@@ -185,6 +185,10 @@ pub struct PaymentMethodResponse {
     #[cfg(feature = "payouts")]
     #[schema(value_type = Option<Bank>)]
     pub bank_transfer: Option<payouts::Bank>,
+
+    #[schema(value_type = Option<PrimitiveDateTime>, example = "2024-02-24T11:04:09.922Z")]
+    #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
+    pub last_used_at: Option<time::PrimitiveDateTime>,
```

Comment:
Not really a blocker, but we should probably discuss whether to send the last_used_at field as an ISO 8601 string or as an Unix timestamp. Personally I'm fine with ISO 8601. Confirm with @tinu-hareesswar.